x-pack/solutions/security/plugins/security_solution/public/flyout/ui_tests/tests/alert_details_url_sync.spec.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, spaceTest } from '@kbn/scout-security';
+import { CUSTOM_QUERY_RULE } from '@kbn/scout-security/src/playwright/constants/detection_rules';
+
+const RIGHT = 'right';
+
+spaceTest.describe('Expandable flyout state sync', { tag: ['@ess', '@svlSecurity '] }, () => {
```

Comment:
What do you think about overriding tags in `kbn/scout-security`:

```
import { tags as defaultTags } from '@kbn/scout';

const SERVERLESS_ONLY = ['@svlSecurity'];

export const tags = {
  ESS_ONLY: defaultTags.ESS_ONLY,
  SERVERLESS_ONLY,
  DEPLOYMENT_AGNOSTIC: SERVERLESS_ONLY.concat(defaultTags.ESS_ONLY),
  PERFORMANCE: defaultTags.PERFORMANCE,
};
```

Reply:
I believe that this might be great, but before putting the implementation in place, it should be discussed with the different area teams, since we tried to do a similar approach for our current Cypress tests and everybody preferred to have all the tags explicitly in the tests.

Reply:
>everybody preferred to have all the tags explicitly in the tests.

Understood. Currently Scout will throw an error if test will use a new tag, e.g. "myFeature". It is not that we don't plan to support custom tags, but we would like to limit it for the moment before we know pros/cons.